app/models/manageiq/providers/redfish/physical_infra_manager/event_catcher.rb

```diff
@@ -0,0 +1,13 @@
+module ManageIQ::Providers::Redfish
+  class PhysicalInfraManager::EventCatcher < ::MiqEventCatcher
+    require_nested :Runner
+
+    def self.ems_class
+      PhysicalInfraManager
+    end
+
+    def self.settings_name
+      :event_catcher_redfish_physical_infra
```

Comment:
If you don't define this it is just `:event_catcher_redfish` right?  That should be fine, unless you have multiple managers under your provider its best to just leave this default.

Reply:
I did this in order to "fit in" with other providers (I took inspiration from Nuage and Lenovo). And I am more than happy to remove a few lines from provider;)